app/views/Posts/show.html.erb

```diff
@@ -1,16 +1,18 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <title>BlogApp</title>
-    <meta name="viewport" content="width=device-width,initial-scale=1">
-    <%= csrf_meta_tags %>
-    <%= csp_meta_tag %>
+<div>
+  <h2>
+    <%= @single_post.title %> by <%= @single_post.author.name %>
+  </h2>
+  <span>
+    Comments: <%= @single_post.comments_counter %>
+    Likes: <%= @single_post.likes_counter %>
+  </span>
+  <p>
+    <%= @single_post.text %>
+  </p>
+</div>
 
-    <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
-    <%= javascript_importmap_tags %>
-  </head>
-
-  <body class="body">
-    <h1>Show method for posts</h1>
-  </body>
-</html>
\ No newline at end of file
+<div class="post-comment">
+  <% @post_comments.each do |comment| %>
+    <span><%= comment.author.name %>: <%= comment.text %></span> 
+  <% end %>
+</div>
```

Comment:
- [ ] As I mentioned in the previous comment, kindly ensure that the `app/views/Posts/show.html.erb` view page is working as it should display this **[wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_single_post.png)**(i.e. Post show page should display post with all comments listed as shown on the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_single_post.png)). Right now it shows the below error, so impossible to check the result:
![Screenshot from 2023-01-27 04-28-21](https://user-images.githubusercontent.com/83042742/214997790-0b55cb9d-8298-4660-b44f-7c8983884336.png)

**In short, any page for "Post" is working, all pages show the similar error. Make sure to fix that**